starknet_py/tests/e2e/docs/code_examples/test_account.py

```diff
@@ -0,0 +1,124 @@
+# pylint: disable=unused-variable
+from unittest.mock import patch
+
+import pytest
+from starkware.starknet.public.abi import get_selector_from_name
+
+from starknet_py.constants import FEE_CONTRACT_ADDRESS
+from starknet_py.net.account.account import Account
+from starknet_py.net.client_models import Call
+from starknet_py.net.full_node_client import FullNodeClient
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.models import StarknetChainId
+from starknet_py.net.models.typed_data import TypedData
+from starknet_py.net.networks import TESTNET
+from starknet_py.net.signer.stark_curve_signer import KeyPair, StarkCurveSigner
+
+
+def test_init():
+    # docs-start: init
+    account = Account(
+        address=0x123,
+        client=GatewayClient(net=TESTNET),
+        signer=StarkCurveSigner(
+            account_address=0x123,
+            key_pair=KeyPair(12, 34),
+            chain_id=StarknetChainId.TESTNET,
+        ),
+    )
+    # or
+    account = Account(
+        address=0x123,
+        client=FullNodeClient(node_url="your.node.url", net=TESTNET),
+        key_pair=KeyPair(12, 34),
+        chain=StarknetChainId.TESTNET,
+    )
+    # docs-end: init
+
+
+@pytest.mark.asyncio
+async def test_execute(gateway_account, contract_address):
+    account = gateway_account
+    # docs-start: execute
+    resp = await account.execute(
+        Call(
+            to_addr=contract_address,
+            selector=get_selector_from_name("increase_balance"),
+            calldata=[123],
+        ),
+        max_fee=int(1e15),
+    )
+    # or
+    # docs-end: execute
+    call1 = call2 = Call(
+        to_addr=contract_address,
+        selector=get_selector_from_name("increase_balance"),
+        calldata=[123],
+    )
```

Comment:
I'm not sure about `call1 = call2` being readable enough.

Reply:
But it is not included in the docs 😅

Reply:
Oh, right. Disregard then.